aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -3,14 +3,24 @@
 
 use crate::utils::*;
 use anyhow::Result;
-use aptos_types::on_chain_config::FeatureFlag;
+use aptos_types::on_chain_config::FeatureFlag as AFeatureFlag;
 use move_model::{code_writer::CodeWriter, emit, emitln, model::Loc};
+use serde::{Deserialize, Serialize};
 
+#[derive(Clone, Deserialize, PartialEq, Eq, Serialize)]
 pub struct Features {
     pub enabled: Vec<FeatureFlag>,
     pub disabled: Vec<FeatureFlag>,
 }
 
+#[derive(Clone, Deserialize, PartialEq, Eq, Serialize)]
+#[allow(non_camel_case_types)]
+#[serde(rename_all = "snake_case")]
+pub enum FeatureFlag {
```

Comment:
So every time we add a new feature flag to the yaml file, we'd need to update this? I don't think it's a problem - just confirming.

Reply:
This is correct. I'm expecting us to update this.